x-pack/plugins/enterprise_search/common/guided_onboarding/search_guide_config.ts

```diff
@@ -11,11 +11,11 @@ import { i18n } from '@kbn/i18n';
 export const searchGuideId = 'search';
 export const searchGuideConfig: GuideConfig = {
   title: i18n.translate('xpack.enterpriseSearch.guideConfig.title', {
-    defaultMessage: 'Search my data',
+    defaultMessage: 'Build search experiences with Elasticsearch',
   }),
   description: i18n.translate('xpack.enterpriseSearch.guideConfig.description', {
     defaultMessage:
-      'Build custom search experiences with your data using Elastic’s out-of-the-box web crawler, connectors, and robust APIs. Gain deep insights from the built-in search analytics to curate results and optimize relevance.',
+      'We’ll help you build a search experience with your data using Elastic’s web crawler, connectors, and APIs.',
```

Comment:
Extremely nitpicky, but you're using a smart quote (`’`) instead of a straight quote (`'`) here in both "We’ll" and "Elastic’s". We should be using straight quotes.